schema/site.schema.json

```
@@ -49,6 +49,27 @@
       "type": "boolean",
       "default": false
     },
+    "RateLimits": {
```

properties in the schema are usually lowercase

```suggestion
    "rateLimits": {
```